src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/IDispatchMetaObject.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CSharp.RuntimeBinder.ComInterop
     internal sealed class IDispatchMetaObject : ComFallbackMetaObject
     {
         private readonly IDispatchComObject _self;
-        private static readonly bool[] s_isByRef = new bool[] { false };
+        private static readonly bool[] s_false = new bool[] { false };
```

Comment:
is this 'describe value instead of semantic meaning of value' thing a BCL code standards rule?

Reply:
I think it's more that when elevated to this level, that's the only meaning it has. It could be reused for other things.

Reply:
Right. As a field, there is no semantic meaning. It describes what it is and then use sites can choose to apply those values wherever they make sense.